internal/scan/scan.go

```diff
@@ -358,6 +358,21 @@ func validateTagLocal(ctx context.Context, tag *v1.TagReference, cfg *types.Conf
 func walkDirScan(ctx context.Context, cfg *types.Config, tag *v1.TagReference, component *types.OpenshiftComponent, mountPath string) *types.ScanResults {
 	results := types.NewScanResults()
 
+	// Check the operating system release against a known list of certified
+	// distributions. Here we're primarily concerned about warning against
+	// operating systems that haven't been certified, yet.
+	osInfo := validations.ValidateOS(cfg, mountPath)
+	osScanResult := types.NewScanResult().SetOS(osInfo).SetComponent(component).SetTag(tag)
+	if component != nil && osScanResult.Error != nil {
+		if i, ok := cfg.PayloadIgnores[component.Component]; ok {
+			if tag != nil {
+				if !i.ErrIgnores.IgnoreTag(tag.Name, osScanResult.Error.Error) {
+					results.Append(osScanResult)
+				}
+			}
+		}
+	}
+
```

Comment:
So, if something is certified, it is not reflected in the results? IOW only errors are shown?

Reply:
>  IOW only errors are shown?

Yes

Reply:
This is not the way the rest of the system works. We are able to look into full logs and see that such and such is verified.

@rphillips WDYT?

Reply:
Fixed.